Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -108,7 +108,7 @@ def setUp(self):
             )
         finder = self.machinery.FileFinder(None)
         self.spec = importlib.util.find_spec(self.name)
-        self.assertIsNotNone(self.spec)
+        assert self.spec
```

Comment:
Why skip this in optimized mode?

Reply:
FYI - I only care about the retention of bare asserts in the tests for importlib.metadata and importlib.resources. Unfortunately, the tests for importlib.metadata are interspersed with other importlib tests, so it's not easy to know which are which without looking at the [cpython branch of importlib_metadata](https://github.com/python/importlib_metadata/tree/cpython). I do have plans to move importlib.metadata tests into their own package so they're clearly separated.